server/routes/prisons/categoryGroups/singleCategoryGroupController.ts

```diff
@@ -12,6 +12,8 @@ export default class SingleCategoryGroupController {
   public view(): RequestHandler {
     return async (req, res) => {
       const { reference, prisonId } = req.params
+      const { sessionTemplateRef } = req.query
```

Comment:
This needs sanitising / validating. Intending to do this separately alongside checking the `:reference` parameter that's used in a lot of routes.